### PR TITLE
SoftVC VITS Singing Voice Conversion tutorial

### PR DESCRIPTION
**Pull request by aleksandr-mokrov**
CVS-121982

### CODE REVIEW

File: notebooks/259-softvc-voice-conversion/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# SoftVC VITS Singing Voice Conversion and OpenVINO™
```
Comment (igor-davidyuk) [inline notebooks/259-softvc-voice-conversion/README.md:1]: does it work in Colab or Binder? If it does please add badges

File: notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb [hunk 2]
```diff
@@ -643,8 +643,7 @@
       "\t\t\tRemoves the SoftMax layer that is the output layer\n",
       "  --remove_memory \n",
       "\t\t\tRemoves the Memory layer and use additional inputs outputs instead\n",
-      "\t\t\t\n",
-      "\n"
+      "\t\t\t\n"
```
Comment (igor-davidyuk) [inline notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb:646]: did you push this by mistake?

File: notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb [hunk 3]
```diff
@@ -0,0 +1,420 @@
+{
```
Comment (igor-davidyuk) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #4.    %pip install tqdm librosa torch torchaudio faiss-cpu gradio "numpy==1.23.5" fairseq "fairseq==0.12.2" praat-parselmouth
</pre><p>-q</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
```
Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <p>Please preserve output of these cells for demonstration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <p>Please preserve output of this cell for demonstration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.        kwargs["raw_audio_path"] = src_audio
</pre><p>Can other values be parametrized?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.        print(audio)
</pre><p>This line seems redundant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <p>Please preserve output of these cells for demonstration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <p>Please preserve output of this cell for demonstration</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #12.        kwargs["raw_audio_path"] = src_audio
</pre><p>Can other values be parametrized?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (itrushkin) [inline notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #14.        print(audio)
</pre><p>This line seems redundant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: In this tutorial there are only audio in the output. We can't play them in the github interface.

File: notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb (hunk 2 above)
Comment (aleksandr-mokrov) [reply notebooks/121-convert-to-openvino/121-convert-to-openvino.ipynb]: Yes, fixed

File: notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb (hunk 3 above)
Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb:1]: Added

File: notebooks/259-softvc-voice-conversion/README.md (hunk 1 above)
Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/README.md]: Only in Colab it works. Binder creates a VM with only 2GB RAM, that is not enough.

File: notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb (hunk 4 above)
Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb]: Some adjustable params were added

Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb]: Removed

Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb]: Removed

Comment (aleksandr-mokrov) [reply notebooks/259-softvc-voice-conversion/259-softvc-voice-conversion.ipynb]: Some adjustable params were added

File: notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb [hunk 5]
```diff
@@ -0,0 +1,440 @@
+{
```
Comment (eaidova) [inline notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: <blockquote>These feature vectors are directly fed into VITS without the need for conversion to a text-based intermediate representation.</blockquote><p>could you please add link on VITS?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (eaidova) [inline notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: <p>I'm not sure that we can mention explicitly My Little Pony in notebook due to trademarks, can this info be committed?</p><p>Better to provide info about how to change to other models in this comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (eaidova) [inline notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: <p>maybe better to add directory in sys.path?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1384/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (aleksandr-mokrov) [reply notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: Added

Comment (aleksandr-mokrov) [reply notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: Changed

Comment (aleksandr-mokrov) [reply notebooks/261-softvc-voice-conversion/261-softvc-voice-conversion.ipynb:1]: It is better, but so-vits-svc project had some hardcoded paths, like `"pretrain/checkpoint_best_legacy_500.pt"` and there is no way to pass it without changing the code.

**Review by eaidova — APPROVED** [review]

**Comment by eaidova** [comment]
@itrushkin do you have any additional comments?

**Review by itrushkin — APPROVED** [review]

**Pull request merged**